src/card.ts

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+import type {
+  Marqeta,
+  MarqetaOptions,
+} from './'
+
+export interface ExpirationOffset {
+  unit?: string;
+  value?: bigint;
+}
+
+export interface AddressType {
+  firstName?: string;
+  middleName?: string;
+  lastName?: string;
+  address1: string;
+  address2?: string;
+  city?: string;
+  state?: string;
+  zip?: string;
+  country?: string;
+  phone?: string;
+  postalCode?: string;
+}
+
+export interface Shipping {
+  method?: string;
+  returnAddress?: AddressType;
+  recipientAddress?: AddressType;
+  carOfLine?: string;
+}
+
+export interface NameLineType {
+  [value: string]: string;
+}
+
+export interface Text {
+  nameLine1: NameLineType;
+  nameLine2?: NameLineType
+  nameLine3?: NameLineType
+}
```

Comment:
I don't think this is the structure you really want... and I don't think the `NameLineType` is either. I think this shouldn't be called `Text`, because it's almost as confusing as if it were called `String`... but the structure should probably be:

```typescript
 export interface Text {
   nameLine1: {
     value: string;
   };
   nameLine2?: {
     value: string;
   };
   nameLine3?: {
     value: string;
   };
 }
```

because the `value` is the name of the key and your structure allows for any string key... and then there's the missing semi-colons on the last two lines...